tests/VirtoCommerce.PriceExportImportModule.Tests/CsvHelperTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using CsvHelper;
+using CsvHelper.Configuration;
+using VirtoCommerce.PriceExportImportModule.Data.Models;
+using VirtoCommerce.PriceExportImportModule.Data.Services;
+using Xunit;
+
+namespace VirtoCommerce.PriceExportImportModule.Tests
+{
+    public class CsvHelperTests
+    {
+
+        [Fact]
+        public async Task TestDoubleBadDataFoundCase()
```

Comment:
Why not `Assert.Equal(2, errorCount);`?

Reply:
DoubleBadDataFound produces only one error, why? Here https://github.com/VirtoCommerce/vc-module-price-export-import/pull/65/files#diff-78278f2491d16515959638326452e5bcce817ea68865b9b70b8dbfcf0eb7b377R277 we have one error per row.